Tryouts/Prototypes/Shell/Node-launcher/Lib/src/BrowserWindow.ts

```diff
@@ -0,0 +1,25 @@
+export class BrowserWindow {
```

Comment:
What's this class for? Is it a stub for future implementation?

Reply:
Yes, I accidentally committed that WIP code after I rearranged the project files for the monorepo.
I've pushed the intended code just now.